src/commands/accesslogs.js

```diff
@@ -34,7 +34,7 @@ export async function accessLogs(params) {
   });
 
   if (format === 'human') {
-    Logger.warn(styleText('yellow', '/!\\ Access Logs feature is in Alpha testing phase'));
+    Logger.warn(styleText('yellow', '/!\\ This feature is in Beta testing phase'));
```

Comment:
To align with other testing phase messages: 

```suggestion
    Logger.println(styleText('yellow', '/!\\ This feature is in Beta testing phase'));
```